src/citra_qt/configuration/config.h

```diff
@@ -33,4 +33,6 @@ class Config {
 
     std::unique_ptr<QSettings> qt_config;
     std::string qt_config_loc;
+
+    std::string GetHostOS();
```

Comment:
```suggestion
    std::string GetHostOS() const;
```
(ditto in the cpp file)

This should also be located under `WriteSetting`'s declaration.

Reply:
By using QString::toStdString(), I could get rid of this helper method. Problem solved :)